spark/src/test/scala/org/apache/spark/sql/delta/InCommitTimestampSuite.scala

```diff
@@ -441,5 +445,401 @@ class InCommitTimestampSuite
       assert(blob("exceptionStackTrace").contains(Snapshot.getClass.getName.stripSuffix("$")))
     }
   }
+
+  test("DeltaHistoryManager.getActiveCommitAtTimeFromICTRange") {
+    withTempDir { tempDir =>
+      spark.range(10).write.format("delta").save(tempDir.getAbsolutePath)
+      val startTime = System.currentTimeMillis()
+      val clock = new ManualClock(startTime)
+      // Ensure that a cached version of deltaLog without the ManualClock is not used.
+      DeltaLog.clearCache()
+      val deltaLog =
+        DeltaLog.forTable(spark, new Path(tempDir.getCanonicalPath), clock)
+      val commitTimeDelta = 10
+      val numberAdditionalCommits = 25
+      assert(clock.getTimeMillis() == deltaLog.clock.getTimeMillis())
```

Comment:
simpler to just do this?
```suggestion
      assert(clock eq deltaLog.clock)
```